docs/_sidebar.md

```diff
@@ -24,16 +24,35 @@
   - [2.5.2 - Kubernetes](2-virtual-machines-containers/container-management_kubernetes.md)
   - [2.5.3 - Deliverables](2-virtual-machines-containers/container-management_deliverables.md)
 * **Chapter 3**
-- [3.0 - Software Development Practices](3-software-development/overview.md)
-- [3.1 - Agile](3-software-development/agile.md)
-- [3.2 - Jira](3-software-development/jira.md)
-- [3.3 - Version Control](3-software-development/version-control.md)
-  - [3.3.1 - Git](3-software-development/version-control_git.md)
-  - [3.3.2 - GitHub](3-software-development/version-control_github.md)
-  - [3.3.3 - Deliverables](3-software-development/version-control_deliverables.md)
-- [3.4 - Pair Programming](3-software-development/pair-programming.md)
-- [3.5 - Testing](3-software-development/testing.md)
-- [3.5 - Hello DevOps](3-software-development/hello-devops.md)
+- [3.0 - Development Practices](3/overview.md)
+- [3.1 - Agile](3/agile.md)
+<<<<<<< HEAD
```

Comment:
Looks like there was a merge conflict that still needs to be resolved.

Reply:
Fixed the merge conflict in the latest commit.